engines/joomla/nucleus/particles/contentarray.html.twig

```diff
@@ -21,7 +21,12 @@
     {% set categories = joomla.finder('category', category_options).published(1).language().limit(0).find() %}
 
     {# Content Finder #}
-    {% set article_finder = joomla.finder('content').category(categories).published(1).language() %}
+    {% if filter.articles %}
+        {% set article_options = filter.articles ? {id: [filter.articles|split(',')]} : {} %}
```

Comment:
Oh I know it's already being parsed and cleaned up at a lower level, but I think it's best to remove the spaces via twig and going with the assumption that we don't really know if they will or will not get removed to whatever final method we pass the data to.
I believe this is just a better practice: pass the data the way you know it should be handled as. This is what I meant by future proof, in the eventuality that for whatever reason the end point is not stripping spaces, this implementation will most likely fail, however if we had it set to never pass spaces in the first place, we know we will be ok.

And I know I'm being picky here, it's just a minor enhancements that I'd like to see added as I know it doesn't  costs much in terms of performance. 😃